src/NSubstitute/NSubstitute.csproj

```diff
@@ -11,32 +11,23 @@
     <PackageProjectUrl>https://nsubstitute.github.io/</PackageProjectUrl>
     <PackageLicenseExpression>BSD-3-Clause</PackageLicenseExpression>
   </PropertyGroup>
+
   <ItemGroup>
     <None Include="..\..\docs\images\nsubstitute-100x100.png" Pack="true" PackagePath="icon.png" Visible="false" />
   </ItemGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;netstandard2.0;net45;net46;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;netstandard2.0;net462</TargetFrameworks>
   </PropertyGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
-    <PackageReference Include="System.Linq.Queryable" Version="4.3.0-*" />
-    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.3.0-*" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3' or '$(TargetIsNetFx)' == 'true'">
-    <PackageReference Include="Castle.Core" Version="4.4.1-*" />
+  <ItemGroup>
+    <PackageReference Include="Castle.Core" Version="5.1.1" />
```

Comment:
I would keep the `PackageReference` version ranges as they were. It was `5.0.0-*` for `Castle.Core`.

Reply:
I restored original versions, but what is wrong with it? why do not use patched versions?

Reply:
Thank you! My point is to use floating/ range versions vs direct versions (`5.1.1`).
As for `5.0.0-*` vs `5.1.1-*`, dependencies are "flattened" and can be managed and upgraded by Application/NuGet directly. AFAIK NSubstitute works well with 5.0.0. I'm not aware of a strong reason to limit the lower boundary. Maybe I'm missing something.

In general, the floating to direct version change and mismatch of direct versions can potentially break an application.